### PR TITLE
TDI-38528:Azure queue standalone components don't work

### PR DESCRIPTION
**Pull request by pyzhou-talend**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

https://jira.talendforge.org/browse/TDI-38528

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/151/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/151/)

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueCreateRuntime.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.talend.daikon.i18n.I18nMessages;
+
+import com.microsoft.azure.storage.StorageErrorCodeStrings;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.queue.CloudQueue;
+
+public class AzureStorageQueueCreateRuntime extends AzureStorageQueueRuntime
+        implements ComponentDriverInitialization<ComponentProperties>{
+    
+    private static final long serialVersionUID = 1L;//?? what's this?
```
Comment (akhabali) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueCreateRuntime.java]: Please refer to http://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html
this is the serialization version of the class. I usually just generate one.
here is a nice article about it : http://www.javapractices.com/topic/TopicAction.do?Id=45

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueDeleteRuntime.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    @Override
+    public void runAtDriver(RuntimeContainer container) {
+        deleteAzureQueue(container);
+        setReturnValues(container);
+    }
+
+    private boolean deleteAzureQueue(RuntimeContainer container) {
+        String queue = QueueName;
+        Boolean startable = false;
```
Comment (akhabali) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueDeleteRuntime.java]: Please rename this to deletionResult or something like that. the **startable** was for the reader purpose.
You may also make this method return void. as the returned value is never used

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueuePurgeRuntime.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            .getI18nMessages(AzureStorageQueuePurgeRuntime.class);
+
+    @Override
+    public void runAtDriver(RuntimeContainer container) {
+        purgeAzureQueue(container);
+        setReturnValues(container);
+    }
+
+    private boolean purgeAzureQueue(RuntimeContainer container) {
+        Boolean startable = false;
```
Comment (akhabali) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueuePurgeRuntime.java]: same thing here. 
please refer to my previous comment

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueCreateRuntime.java (hunk 1 above)
Comment (pyzhou-talend) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueCreateRuntime.java]: Ok. It's very helpful.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/158/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/158/)

**Pull request merged**